### PR TITLE
run/system_package: add e2e test coverage and update dockerfile

### PR DESCRIPTION
**Pull request by grayside**
* Remove skips
* Update system package Dockerfile to more closely align with revised standard
* Add e2e test coverage to system package sample

### CODE REVIEW

File: run/system_package/e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	if err := service.Deploy(); err != nil {
+		t.Fatalf("service.Deploy %q: %v", service.Name, err)
+	}
+	defer service.Clean()
+
+	requestPath := "/diagram.png"
+	req, err := service.NewRequest("GET", requestPath)
+	if err != nil {
+		t.Errorf("service.NewRequest: %q", err)
+		return
```
Comment (tbpg) [inline run/system_package/e2e_test.go]: `Fatalf` instead of `Errorf` and return?

File: run/system_package/e2e_test.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	}
+	q := req.URL.Query()
+	q.Add("dot", "digraph G { A -> {B, C, D} -> {F} }")
+	req.URL.RawQuery = q.Encode()
+
+	client := http.Client{Timeout: 10 * time.Second}
+	resp, err := client.Do(req)
+	if err != nil {
+		t.Errorf("client.Do: %v", err)
+		return
```
Comment (tbpg) [inline run/system_package/e2e_test.go]: Same comment.

**Comment by tbpg** [comment]
Sorry for the slow review! Looks good!

**Comment by grayside** [comment]
Failed on a flake. Updating branch.

**Comment by tbpg** [comment]
Flaky test (#1094) failed. Merging this anyway since it's unrelated.

**Pull request merged**